src/Symfony/Component/Translation/Translator.php

```diff
@@ -174,6 +202,13 @@ public function trans(?string $id, array $parameters = [], ?string $domain = nul
             }
         }
 
+        if ($this->globalTranslatableParameters && !isset($this->globalParameters[$locale])) {
+            $this->globalParameters[$locale] = []; // Avoid infinite loops
```

Comment:
the storage of resolved global parameters per locale should be separate from the storage of the user-provided configuration IMO, instead of using a `*` key.

Reply:
Done using a `globalTranslatedParameters`.